PA-12031 Add missing docs CSA, SCA, SBOM (#12)
soos-io/kb-docs: CSA/AzureDevOps.md

> # How to Integrate SOOS CSA with your Azure CI

Reviewer: ```suggestion
# How to Integrate SOOS CSA with your Azure DevOps CI Pipeline
```

<div>
<img src="../assets/img/SOOS-Icon.png" alt="SOOS" width="128" height="128">
<img src="../assets/img/azure.png" alt="Azure" width="128" height="128">
</div>

> This document will take you step-by-step through the tasks required to set up an Azure project, for scan it with the SOOS CSA Product.

Reviewer: ```suggestion
This document will take you step-by-step through the tasks required to set up an Azure project and scan it with SOOS Container Analysis.
```

## Prerequisites

- You need to have a [SOOS account.](https://app.soos.io/register)
- You need to have a Azure Project configured.

## Steps

### **Installing the task**

The SOOS task will be found under the shared extension listing in your organization settings. Click the option to install the task. The installation has been completed successfully when it is visible under the Installed listing.

<img src="../assets/img/azure-install.png">

### **Setup variables**

> Once installed search for the SOOS under tasks and proceed with the configuration.

Reviewer: ```suggestion
Once installed search for SOOS under tasks and proceed with the configuration.
```

<img src="../assets/img/azure-task.png">

- Select Container Security Analysis (CSA) for the Scan Type parameter

> - Configure the SOOS variables, either directly in the yaml section or in the Task variables section.  Use the API Key and Client ID values you collected from the SOOS App.

Reviewer: ```suggestion
- Configure the SOOS variables, either directly in the yaml section or in the Task variables section. Use the API Key and Client ID values you found on the SOOS App integration page.
```
(all markup link to integrate page too)

> Make sure to also set the Display Name, Project Name (which groups scans together), Target Uri and Scan mode parameters.

Reviewer: ```suggestion
Be sure to set the Display Name, Project Name (which groups scans together), Target Uri and Scan mode parameters.
```

Reviewer: ^ that looks like DAST

<img src="../assets/img/csa-azure-variables.png">

### **Setting up inside your pipeline.yml**

> Once we have defined these variables globally you can set up the task to be used inside your pipeline.yml following this example script.

Reviewer: ```suggestion
Once these variables have been defined globally, you can set up the Task to be used inside your pipeline.yml following this example script.
```
